EncoreTickets.SDK/Api/ApiClientWrapperBuilder.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+using EncoreTickets.SDK.Api.Context;
+using EncoreTickets.SDK.Api.Helpers;
+using EncoreTickets.SDK.Api.Helpers.RestClientWrapper;
+
+namespace EncoreTickets.SDK.Api
+{
+    internal static class ApiClientWrapperBuilder
+    {
+        public static RestClientWrapper CreateClientWrapper(ApiContext context)
+        {
+            var credentials = context == null
+                ? null
+                : new RestClientWrapperCredentials
+                {
+                    AuthenticationMethod = context.AuthenticationMethod,
+                    AccessToken = context.AccessToken,
+                    Username = context.UserName,
+                    Password = context.Password
+                };
+            return new RestClientWrapper(credentials);
+        }
+
+        public static RestClientParameters CreateClientWrapperParameters(ApiContext context, string baseUrl, string endpoint,
+            RequestMethod method, object body)
+        {
+            return new RestClientParameters
+            {
+                BaseUrl = baseUrl,
+                RequestUrl = endpoint,
+                RequestBody = body,
+                RequestFormat = RequestFormat.Json,
+                RequestHeaders = GetHeaders(context),
+                RequestMethod = method,
+                RequestQueryParameters = GetQueryParameters(context, method),
+                RequestUrlSegments = null,
+            };
+        }
+
+        private static Dictionary<string, string> GetHeaders(ApiContext context)
+        {
+            var headers = new Dictionary<string, string>
+            {
+                {"x-SDK", "EncoreTickets.SDK.NET" } // todo: add build numbers
```

Comment:
Can you create a ticket to fix that TODO and delete this comment?

Reply:
I am going to remove this when I will work with a ticket with the addition of this library to Nuget. Is it ok?

Reply:
Well, if you have a ticket already you can mention it here. That's just a way to work with bus factor.